Makefile

```diff
@@ -319,6 +319,10 @@ $(GOPATH)/bin/go-junit-report:
 test-results/junit.xml: $(GOPATH)/bin/go-junit-report test-results/test.out
 	cat test-results/test.out | go-junit-report > test-results/junit.xml
 
+.PHONY: printtestresultannotations
+printtestresultannotations:
```

Comment:
```suggestion
print-test-result-annotations:
```
?

Reply:
this is super hard to type, I will change to `test-report` (like Maven has `mvn test:report`